aquarium.py

```diff
@@ -30,9 +32,9 @@
 REVAULTD_SRC_DIR = os.path.join(SRC_DIR, "revaultd")
 REVAULT_GUI_SRC_DIR = os.path.join(SRC_DIR, "revault-gui")
 SHELL = os.getenv("SHELL", "bash")
-COORDINATORD_VERSION = os.getenv("COORDINATORD_VERSION", "9b90fcd3ad5dfb51bf50dcaef859c5cc00dbfe3a")
-COSIGNERD_VERSION = os.getenv("COSIGNERD_VERSION", "e41e5cafc33568f61076700ba360f831e3b072cc")
-REVAULTD_VERSION = os.getenv("REVAULTD_VERSION", "gethistory-2")
+COORDINATORD_VERSION = os.getenv("COORDINATORD_VERSION", "master")
+COSIGNERD_VERSION = os.getenv("COSIGNERD_VERSION", "master")
+REVAULTD_VERSION = os.getenv("REVAULTD_VERSION", "master")
 REVAULT_GUI_VERSION = os.getenv("REVAULT_GUI_VERSION", "gethistory")
```

Comment:
you can update on `revault:master` now

Reply:
Let's do that in a followup PR to not have to rebase everything